src/utils/expressionColors.ts

```diff
@@ -0,0 +1,326 @@
+export const expressionColors = {
```

Comment:
if we're defining expressionColors ourselves in this repo, why not convert all the keys to camelcase instead of adding the mapping function?

Reply:
done